input_bag_path: /home/he/Data/truck/12-30-truck-model/12.bag  #your bag

# output_path: same as input path

extrinsic_path: /home/he/Code/Git/MyRepo/load_data_ws/src/load_rosbag_data/data/   #transformation, usually Identity

num_lidar_points: 50  #save frames

message_title: /livox/lidar_all #message topic